.github/workflows/build.yml

```
@@ -382,7 +458,22 @@ jobs:
             --build \
             --parallel \
             --build_shared_lib \
-            "${build_opts[@]}"
+            "${build_opts[@]}" \
+            1>${{ env.ONNXRUNTIME_NAME == 'onnxruntime' && '&1' || format('{0}_stdout.txt', matrix.artifact_name) }} \
+            2>${{ env.ONNXRUNTIME_NAME == 'onnxruntime' && '&2' || format('{0}_stderr.txt', matrix.artifact_name) }}
+
+      - name: Send build log to Discord
+        if: failure() && steps.build.outcome == 'failure' && env.ONNXRUNTIME_NAME == 'voicevox_onnxruntime'
+        run: |
+          curl \
+            "$DISCORD_WEBHOOK" \
+            -sf \
+            -F 'payload_json={"username":"onnxruntime-builder"}' \
+            -F file1=@${{ matrix.artifact_name }}_stdout.txt \
+            -F file2=@${{ matrix.artifact_name }}_stderr.txt \
+            > /dev/null
+        env:
+          DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}

```

あっすみません！！
`DISCORD_WEBHOOK`だとなんか事故りそうな予感がしたので、`PRODUCTION_DISCORD_WEBHOOK`に変更お願いしても良いでしょうか 🙇 

```suggestion
      - name: Send build log to Discord
        if: failure() && steps.build.outcome == 'failure' && env.ONNXRUNTIME_NAME == 'voicevox_onnxruntime'
        run: |
          curl \
            "$PRODUCTION_DISCORD_WEBHOOK" \
            -sf \
            -F 'payload_json={"username":"onnxruntime-builder"}' \
            -F file1=@${{ matrix.artifact_name }}_stdout.txt \
            -F file2=@${{ matrix.artifact_name }}_stderr.txt \
            > /dev/null
        env:
          PRODUCTION_DISCORD_WEBHOOK: ${{ secrets.PRODUCTION_DISCORD_WEBHOOK}}

```

---

fixup: [`7f0d878` (#52)](https://github.com/VOICEVOX/onnxruntime-builder/pull/52/commits/7f0d87879e2ac0fbcc641754864f8ec0be370dfd)